src/org/ods/services/OpenShiftService.groovy

```diff
@@ -429,7 +458,12 @@ class OpenShiftService {
         )
     }
 
-    // Returns data about the pods (replicas) of the deployment.
+    boolean isDeploymentKind(String kind) {
```

Comment:
This feels like a private method

Reply:
It used when generating the TIR data in assembleDeployments called by org.ods.orchestration.usecase.LeVADocumentUseCase#createTIR
My rationale was that the createTIR method should not need to know what deployment kinds we support. Let's discuss more.